packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions/components/InputGroup/components/QuestionInput.spec.js

```diff
@@ -0,0 +1,259 @@
+import { expect } from 'chai'
+import { Grommet } from 'grommet'
+import React from 'react'
+import sinon from 'sinon'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import { QuestionInput } from './QuestionInput'
+
+describe('Component > QuestionInput', function () {
+  describe('with checkbox type', function () {
+    it('should render without crashing', function () {
+      render(
+        <Grommet
+          theme={zooTheme}
+          themeMode='light'
+        >
+          <QuestionInput
+            option={{
+              label: 'Eating',
+              value: 'TNG'
+            }}
+            questionId='WHTBHVRSDS'
+            theme={zooTheme}
```

Comment:
This isn't necessary since `Grommet` is wrapping the component and is injecting the theme into context and if you import the decorated component using `withTheme` it then gets passed down as a prop for you.
```suggestion
```